resources/migrations/20220317055918-create-question.down.sql

```diff
@@ -0,0 +1 @@
+DROP TABLE IF EXISTS question;
```

Comment:
Ditto the comment about IF NOT EXISTS in the create migration. Although, it is much less harmful here.